libs/features/mas/docs/merch-card.html

```diff
@@ -145,6 +145,12 @@ <h3 id="attributes" tabindex="-1">Attributes <a class="header-anchor" href="#att
 <td></td>
 <td><code>false</code></td>
 </tr>
+<tr>
+<td><code>daa-lh</code></td>
+<td>Analytics identifier of a card. Value is coming from the ‘PRODUCT_CODE’ tag set on the card (for now manual authoring required, later will be tagged automatically). Sample values: ‘ccsn’ or ‘phlt’.</td>
+<td></td>
+<td><code>false</code></td>
+</tr>
 </tbody>
 </table>
 <h4 id="active-variants%3A" tabindex="-1">Active variants: <a class="header-anchor" href="#active-variants%3A" href="#active-variants%3A" title="Permalink to this heading">#</a></h4>
```

Comment:
not sure where that double href is coming from (again, not your PR)

Reply:
double hrefs?